mlflow/pipelines/utils/execution.py

```diff
@@ -400,6 +400,28 @@ def _create_makefile(pipeline_root_path, execution_directory_path, template) ->
 
     if template == "regression/v1":
         makefile_to_use = _REGRESSION_MAKEFILE_FORMAT_STRING
+        steps_folder_path = os.path.join(pipeline_root_path, "steps")
+        if not os.path.exists(steps_folder_path):
+            os.mkdir(steps_folder_path)
+        for required_file in [
+            "ingest.py",
+            "split.py",
+            "train.py",
+            "transform.py",
+            "custom_metrics.py",
+        ]:
+            required_file_path = os.path.join(steps_folder_path, required_file)
+            if not os.path.exists(required_file_path):
+                try:
+                    with open(required_file_path, "w"):
+                        pass
```

Comment:
```suggestion
                    with open(required_file_path, "w") as f:
                        f.write("# Created by MLflow Pipeliens\n")
```

How about adding a comment in the file to clarify what created this file?